doc/monitoring/metrics_reference.rst

```diff
@@ -708,3 +708,184 @@ Event loop tx thread information:
             -   Event loop prolog time (ms)
         *   -   ``tnt_ev_loop_epilog_time``
             -   Event loop epilog time (ms)
+
+Memtx
+-----
+
+Memtx mvcc memory statistics.
+Transaction manager consists of two parts:
+- the transactions themselves (TXN section)
+- MVCC
+
+..  _metrics-reference-memtx_txn:
+
+TXN
+~~~
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   - ``tnt_memtx_tx_tnx_statements`` are the transaction statements.
```

Comment:
What `tx_tnx` should mean?

Reply:
I decided that there might be another call like `box.stat.memtx.tx()` , `box.stat.memtx.SOMETHING()`. So like breadcrumps:
`box.stat.memtx.tx().txn > memtx_tx_txn `. But yeah, simplifing to `memtx_txn` for transactions and `memtx_mvcc` for MVCC more clearly.

Reply:
Don't forget to update docs.